src/routes/auth-route.ts

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const authRouter = express.Router();
+import {AuthController} from "../controllers/auth-controller";
+
+const authController = new AuthController();
+authRouter.post('/registerUser', authController.registerUser)
+authRouter.post('/login', authController.login)
```

Comment:
Think the router > controller > service is unncessarily verbose - I would put the controller code in here.